### ISSUE TITLE
Update Rancher additional tests to default to the default Rancher branch

### ISSUE THREAD

**Issue by floatingman** [rancher/qa-tasks#1089]
To better facilitate testing Rancher, it is beneficial to have the tests in Jenkins default to a more recent branch of Rancher.

To facilitate this all additional Python checks Jenkins Jobs will be updated to default Rancher branch

**Comment by sowmyav27** [comment]
@floatingman Please confirm with the team if it should be 2.8-head vs 2.9-head before you get a PR up.

**Comment by floatingman** [comment]
@sowmyav27 Sure, this was an older ticket when v2.8 was the default branch. We had a lot of Jenkins jobs that were defaulting to v2.6. I'll reword it to say point to the new default branch

**Comment by rancher-max** [comment]
This was completed a while back and may no longer be necessary, so closing
